ext/pgsql/pgsql.c

```diff
@@ -3131,19 +3022,11 @@ PHP_FUNCTION(pg_last_oid)
 	}
 
 	pgsql_result = pg_result->result;
-#ifdef HAVE_PQOIDVALUE
 	oid = PQoidValue(pgsql_result);
 	if (oid == InvalidOid) {
 		RETURN_FALSE;
 	}
 	PGSQL_RETURN_OID(oid);
-#else
-	Z_STRVAL_P(return_value) = (char *) PQoidStatus(pgsql_result);
```

Comment:
I don't think this code ever compiled with PHP 7, so requiring libpq 7.4 seems to be a no-brainer.